subscription/src/subscription_worker.py

```diff
@@ -52,12 +55,15 @@ def process_messages(sns_client, topic, messages, access_control):
 
             if(access_control.has_read_permission(subscriber, collection_concept_id)):
                 logger.debug(f"Subscription worker: {subscriber} has permission to receive granule notifications for {collection_concept_id}")
+                message_msg = search.process_message(message_body['Message'])
+                message_body['Message'] = message_msg
+                message['Body'] = message_body
                 sns_client.publish_message(topic, message)
             else:
                 logger.info(f"Subscription worker: {subscriber} does not have read permission to receive notifications for {collection_concept_id}.")
```

Comment:
is this an info? it has value to someone in a support role in answering why someone can not get their subscriptions. I'm suggesting warning.

Reply:
I have changed the logging level to warning.